source/css/sass/mixins/_grid.scss

```diff
@@ -0,0 +1,50 @@
+@import "../variables/grid";
+@import "../variables/font";
+@import "../vendor/sass-rem/rem";
+@import "../vendor/sass-mq/mq";
+@import "../functions/grid";
+
+/*
+The grid comprises a central 12 columns plus one left gutter and one right gutter. The central 12 columns are where
+everything normally goes, the side gutters provide centring, and allow elements to appear to break out of the grid.
+The 'breaking out' is to one or both viewport edges, there's no more finely grained differential offset at the moment.
+This could be provided by the addition of extra columns either side of the central 12 columns, but would require
+further investigation to maintain the centring capability provided by the left/right gutters when additional
+columns within the gutters are added.
+*/
+
+// Span every column track
+@mixin grid-span-all {
+  grid-column: 1 / -1;
+}
+
+// Span every main column track (i.e. all except the respective left / right gutter tracks)
+@mixin grid-span-full {
```

Comment:
`grid-span-main`?

Reply:
Did consider that, but I wonder if there's a danger of that being perceived to be deliberately coupled to a `<main>` element. That said, there's probably more danger of `-all` / `-full` confusion, so happy to accept `grid-span-main` if no better name arises.